edb/pgsql/delta.py

```diff
@@ -2155,10 +2170,16 @@ def apply(
             and not context.is_creating(subject)
         ):
             op = self.enforce_constraint(
-                constraint, schema, context, self.source_context)
+                constraint, schema, self.source_context
+            )
+
+            # XXX: PostCommand or maybe even pg_ops have incorrect type
+            # annotations, but I cannot figure them out.
+            op2: sd.Command = op  # type: ignore
```

Comment:
@msullivan

What is the difference between:
- `schema.delta.Command` and
- `pgsql.dbops.Command`?

`sd.Command` gets produced when we diff two schemas? And is then compiled to `pgsql.dbops.Command`?

----

`pgsql.delta.MetaCommand` has a field `pgops`, which collects all operations to be executed.

It should contain `pgsql.dbops.Command`, right? And not `sd.Command`?

Reply:
If I was correct, this should work:
```diff python
 class MetaCommand(sd.Command, metaclass=CommandMeta):
+    pgops: ordered.OrderedSet[dbops.Command]

```

Reply:
A `dbops.Command` represents some concrete postgres DDL command. A `delta.Command` represents a change between edgedb schemas and/or an edgedb DDL command.

I think that `pgops` should probably be just `dbops.Command`s, but I'm not 100% sure what the situation is

Reply:
Cool, i'll continue this in #5672